streamingpro-mlsql/src/main/java/streaming/core/datasource/impl/MLSQLDelta.scala

```diff
@@ -67,12 +75,27 @@ class MLSQLDelta(override val uid: String) extends MLSQLBaseFileSource with WowP
     }
 
     val dataLake = new DataLake(config.df.get.sparkSession)
-    val finalPath = if (dataLake.isEnable) {
-      dataLake.identifyToPath(config.path)
-    } else {
-      resourceRealPath(context.execListener, Option(context.owner), config.path)
+    var loadPath = resourceRealPath(context.execListener, Option(context.owner), config.path)
```

Comment:
This piece of code is a little different, one is in the copywriting, and the other is to avoid the path mode polluting the table location mode when saving.